docs/modules/configure/pages/environment_variables.adoc

```diff
@@ -0,0 +1,30 @@
+= Environment Variables
+
+We try to let most of the things defined through Environment Variables, as https://12factor.net/config[12factor applications's config]. You'll need to configure through your hosting provider (for example in Heroku), through a gem like `figaro` or through docker-compose `env_file`.
```

Comment:
```suggestion
We try to let most of the things defined through Environment Variables, as https://12factor.net/config[12factor applications's config]. You'll need to configure them your hosting provider (for example in Heroku), through a gem like `figaro` or through docker-compose `env_file`.
```

Reply:
I guess `them` was meant to be added after the word `through`, rather replace it? @slickepinne

Reply:
Yeah. 👍